classes/genre.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'item'
+
+# MusicAlbum
+class Genre
+  attr_accessor :name
+  attr_reader :items
+
+  def initialize(name)
+    @id = rand(1..1000)
+    @name = name
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item
```

Comment:
Please consider adding `self` as property of the item object, this is part of the
 [project requirement](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md) see the link for more details on this  [click here >>](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things_divided.md)